server/scripting/script_server.cpp

```
@@ -275,6 +280,29 @@ static void script_server_code_save(struct section_file *file)
   }
 }
 
+/**
+   Dump the bytecode of function passed and send to client to execute.
+ */
+static sol::object execute_client_lua(sol::this_state s, int player,
+                                      sol::function f)
+{
+  sol::state_view lua(s);
+  struct player *pplayer = player_by_number(player);
+  if (pplayer == nullptr) {
+    return sol::make_object(lua, "player not found");
```

Can we raise a lua error instead? (same on line 295)